### DISCUSSION TITLE
Setting default country

### DISCUSSION THREAD

**Discussion by CliffordIsaboke** [bagisto/bagisto#7034] [General]
How do I set a default country and city. This is my code snippet.
`},

                        address: {
                            billing: {
                                address1: [''],
                                save_as_address: false,
                                use_for_shipping: true,
                                country: 'KE'
                            },

                            shipping: {
                                address1: ['']`

**Comment by Amitk-webkul** [comment]
@CliffordIsaboke To set a default country and city in your code snippet, you'll need to specify the default values for the country and city fields in your JavaScript object.
